source/frontend/src/features/properties/list/PropertyListView.scss

```diff
@@ -7,6 +7,8 @@ $contentheading-height: 6rem;
   flex-direction: column;
   flex-grow: 1;
   padding: 0;
+  overflow: hidden;
```

Comment:
This fixes a horizontal scroll bar currently being shown on the property list view:

![image](https://github.com/user-attachments/assets/fe8d6a49-bc78-4b84-bf86-cf8bfcf78ca3)